common/templates/xblock_v2/xblock_iframe.html

```diff
@@ -156,7 +156,7 @@
 <!-- The default stylesheet will set the body min-height to 100% (a common strategy to allow for background
   images to fill the viewport), but this has the undesireable side-effect of causing an infinite loop via the onResize
   event listeners below, in certain situations.  Resetting it to the default "auto" skirts the problem.-->
-<body style="min-height: auto">
+<body style="min-height: auto; background-color: white;">
```

Comment:
😁 Ah, there's where that background was coming from...

Reply:
@bradenmacdonald  Thanks for fixing that There's one tiny spot where that background #f5f5f5 is still being shown: video blocks.

![image](https://github.com/user-attachments/assets/facc832f-4b94-4483-ade3-36f85b81d299)

![image](https://github.com/user-attachments/assets/6ae20fd9-7167-4511-a20f-b9b6b6f0f927)